server/src/dtos/tag.dto.ts

```diff
@@ -55,6 +56,7 @@ export class TagResponseDto {
 export function mapTag(entity: TagEntity): TagResponseDto {
   return {
     id: entity.id,
+    parentId: entity.parentId ?? undefined,
```

Comment:
I'm not sure what you are asking exactly. The response has `parentId` as optional so it will be omitted if it can be omitted if it is `null`. This maps null to undefined so it will be omitted.

Reply:
I meant adding the parentId field here in general. I don't know if this is used in the frontend.

Reply:
I don't think it is used, no. I did need this to validate that the tag hierarchy was correct in the e2e tests though.